java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -1933,13 +1941,6 @@ class DatabaseImpl(
             TABLE_TEXT_PRIMITIVES,
             TABLE_NUMBER_PRIMITIVES
         )
-        private val TABLES_VERSION_2 = TABLES_VERSION_1
-        private val TABLES_VERSION_3 = TABLES_VERSION_2
-        private val TABLES_VERSION_4 = TABLES_VERSION_3
-        private val TABLES_VERSION_5 = TABLES_VERSION_4
-
-        /* internal */
-        val TABLES = TABLES_VERSION_5
 
         private val CREATE_VERSION_3 =
```

Comment:
should we be updating the name of this to CREATE_VERSION_6?

Reply:
Maybe I need to create a new create version, adding @jasonwyatt to take a look at the db migration